xfbin/structure/br/br_xfbinnew.py

```diff
@@ -0,0 +1,379 @@
+from typing import Dict, List, Tuple
```

Comment:
Is this file being used?

Reply:
This one wasn't used, it caused an issue when you attempt to export a model from blender, so instead I used an older version of the file until the issue is fixed. I removed it in the latest commit and added the changes you made back,